src/components/geo/RegionMap/index.js

```diff
@@ -67,6 +67,9 @@ const polygonSourceOptions = {
     promoteId: 'code',
 };
 
+// Paired returns quantitative colors from colorbrewer
+const getRandomColor = () => colors.Paired[12][Math.floor(Math.random() * 12)];
```

Comment:
```suggestion
const getRandomColor = () => getRandomFromList(colors.Paired[12]);
```

getRandomFromList is in fujs